content/docs/sidebar.json

```diff
@@ -60,7 +60,8 @@
         }
       },
       {
-        "slug": "experiments",
+        "slug": "experiments-trail/experiments",
```

Comment:
slug defines URL which looks strange now to my mind (redundant `experiments`)

Reply:
experiments trail will also include params/metrics and the checkpoints as separate sections. this one is about the experiments proper. 

I can rename it.

Reply:
For the time being we can replace the current GS:Experiments page with this one. After #2856 and #2857 are done, we can rename these to "trails" with multiple links among them. 

Naming this now as "trail" will look like a trail with a single stepping stone.

Reply:
sounds good to me!